### DISCUSSION TITLE
Many projects at same time with DDEV

### DISCUSSION THREAD

**Discussion by ophiocus** [ddev/ddev#6769] [Q&A] [answered]
I have the need to manage and develop on a portfolio of over 700 individual, distinct, non transferable, projects.

All explanations I see on DDEV are about "developer work on local on only one project".

I dont want to run _all_ projects concurrently on my machine, but I need to be able to swap the running project at the request of a ticket or an emergency, or a new incoming feature request.

Is DDEV the right tool for this? and if so what makes it the best? I cant find any online references or articles titled "So you have more than one project and want to develop with DDEV"

Is DDEV an industrial grade solution or nay?

**Comment by nickchomey** [comment]
> All explanations I see on DDEV are about "developer work on local on only one project".

Where did you get this, and the other concerns, from?

There's nothing about ddev that suggests it can only be used for one project. It's actually got many features meant for managing multiple projects. And being command line based, you could surely automate a lot of stuff (eg syncing projects, creating ddev config etc) with bash or other scripts 

Just clone each project's directory and db in to a master ddev directory, enter each one and create a ddev config and type ddev start...

**Reply by rfay** [reply, marked as answer]
As @nickchomey says, most folks using DDEV have many projects at the same time. I usually have several running at any given time, and many more stopped. DDEV has always had this feature and it's a favorite of many folks. 

When you access a project with a URL like `https://example.ddev.site` the ddev-router gets that URL and determines form the hostname which project to send it to, so reverse-proxies it to the proper project and webserver. You can read more about this in the docs, https://ddev.readthedocs.io/en/stable/users/usage/architecture/#container-architecture
